cmd/cosign/cli/options/oidc.go

```diff
@@ -40,4 +41,7 @@ func (o *OIDCOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&o.ClientSecret, "oidc-client-secret", "",
 		"[EXPERIMENTAL] OIDC client secret for application")
+
+	cmd.Flags().StringVar(&o.RedirectURI, "oidc-redirect-uri", "",
+		"[EXPERIMENTAL] OIDC redirect URI")
```

Comment:
Related to a comment Bob left on the sigstore/sigstore PR, do you think we should enforce that the port is > 1024? I'd prefer to not encourage Cosign to be run with elevated privileges. Is there a use-case for users providing a redirect with a port that's <1024?

Reply:
I could imagine, you could pass `http://localhost:0/myroute/callback`